app/models/post.rb

```diff
@@ -49,6 +49,8 @@ class Post < ApplicationRecord
   belongs_to :spoiled_unit, polymorphic: true
   has_many :post_likes, dependent: :destroy
   has_many :comments, dependent: :destroy
+  # TODO: what happens if the original is destroyed?
```

Comment:
Reblogs don't have separate comments or likes, so it's really just "do we remove the reblog itself" and the answer is, I think, "hell yes"